src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -35,71 +40,171 @@ public static class Sse
+        public static Vector128<float> CompareNotEqualScalar(Vector128<float> left, Vector128<float> right) => CompareNotEqualScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpngt_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareNotGreaterThan(Vector128<float> left, Vector128<float> right) => CompareNotGreaterThan(left, right);
 
+        /// <summary>
+        /// __m128 _mm_cmpngt_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> CompareNotGreaterThanScalar(Vector128<float> left, Vector128<float> right) => CompareNotGreaterThanScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpnge_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareNotGreaterThanOrEqual(Vector128<float> left, Vector128<float> right) => CompareNotGreaterThanOrEqual(left, right);
 
+        /// <summary>
+        /// __m128 _mm_cmpnge_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> CompareNotGreaterThanOrEqualScalar(Vector128<float> left, Vector128<float> right) => CompareNotGreaterThanOrEqualScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpnlt_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareNotLessThan(Vector128<float> left, Vector128<float> right) => CompareNotLessThan(left, right);
 
+        /// <summary>
+        /// __m128 _mm_cmpnlt_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> CompareNotLessThanScalar(Vector128<float> left, Vector128<float> right) => CompareNotLessThanScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpnle_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareNotLessThanOrEqual(Vector128<float> left, Vector128<float> right) => CompareNotLessThanOrEqual(left, right);
 
+        /// <summary>
+        /// __m128 _mm_cmpnle_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> CompareNotLessThanOrEqualScalar(Vector128<float> left, Vector128<float> right) => CompareNotLessThanOrEqualScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpord_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareOrdered(Vector128<float> left, Vector128<float> right) => CompareOrdered(left, right);
 
+        /// <summary>
+        /// __m128 _mm_cmpord_ss (__m128 a,  __m128 b)
+        /// </summary>
+        public static Vector128<float> CompareOrderedScalar(Vector128<float> left, Vector128<float> right) => CompareOrderedScalar(left, right);
+
         /// <summary>
         /// __m128 _mm_cmpunord_ps (__m128 a,  __m128 b)
         /// </summary>
         public static Vector128<float> CompareUnordered(Vector128<float> left, Vector128<float> right) => CompareUnordered(left, right);
-        
```

Comment:
These files all contain a bunch of lines that are just whitespace... We should probably cleanup separately.